docs/user-guide/kubectl-cheatsheet.md

```diff
@@ -5,22 +5,54 @@ assignees:
 - krousey
 
 ---
-An assortment of compact kubectl examples
+A cheat sheet for `kubectl`.
```

Comment:
How about we remove the line completely.  To me a cheat sheet needs to be terse.